### DISCUSSION TITLE
Using API to get automated county based alerts for OBS

### DISCUSSION THREAD

**Discussion by WeathermanJake** [weather-gov/api#548] [Q&A]
Hi! 

I am writing this because I am stuck in my process of trying to obtain an automated system to have a scrolling text in OBS displaying alerts for certain counties. I've gotten far enough as to know I need the  https://api.weather.gov/alerts/active/zone/{zoneId} link but every time I enter the 3 digit county code, it throws back an error/doesn't accept it. I am not sure if I am using the wrong type of zoneID or what, so any insight is greatly appreciated. 

For reference, this is the county I am trying to get alerts for. 

KY|071|LMK|Warren|KY071|Warren|21227|C|sc|36.9936|-86.4239

**Comment by rr0ss0rr** [comment]
I’m not at my computer right now, but I think you are missing the ‘C’ for county. Should look like KYC071

**Comment by JJKraw** [comment]
All the info you need should be at https://www.weather.gov/documentation/services-web-api  Click on the Specification tab and scroll way down to Zone. You need a full UGC identifier as rr0ss0rr notes. Six characters:

Two letter state code
One letter "C" for county or "Z" for zone
Three digit code

If you are fetching a lot of counties within a state, it's more efficient to do a fetch of /alerts/active/area/{state abbreviation} (but more code to go through the returned JSON).
